### PR TITLE
Fisting reborn

### PR DESCRIPTION
**Pull request by StTargaryen**
Create file

### CODE REVIEW

File: src/HtmlParser.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class HtmlParser {
+  public static void main(String[] args) {
+    System.out.println("hehe");
```
Comment (StTargaryen) [inline src/HtmlParser.java:3]: Hoho

**Review by imaginethis20 — CHANGES_REQUESTED** [review]
123

**Review by StTargaryen — COMMENTED** [review]
123

**Comment by StTargaryen** [comment]
Merge from dev

**Pull request merged**